src/Acre.ts

```diff
@@ -141,7 +141,7 @@ export default class Btc {
   /**
    * You can sign a message according to the Bitcoin Signature format and retrieve v, r, s given the message and the BIP 32 path of the account to sign.
```

Comment:
Bitcoin to replace

Reply:
This one does not need to be changed since it is referring to the generic Bitcoin Signature format, that is not specific to Acre.